lintreview/tools/pep8.py

```diff
@@ -28,9 +28,17 @@ def process_files(self, files):
         to save resources.
         """
         log.debug('Processing %s files with %s', files, self.name)
+        pep8_options = ['exclude',
+                        'filename',
+                        'select',
+                        'ignore',
+                        'max-line-length']
         command = ['pep8', '-r']
         for option, value in self.options.items():
-            command += ' --{}={}'.format(option, value)
+            if option in pep8_options:
+                command += ' --{}={}'.format(option, value)
```

Comment:
Looks like there are potential `UnicodeError` here. The format string should probably be `u' --{}={}'` instead.